dbCodegen/table_base_funcs_gen.go

```diff
@@ -56,7 +56,7 @@ func (g *Generator) buildTableBaseFuncs(table DbTable, importList []string) (str
 
 	// Reverse references
 	for _, rFkey := range table.RevFKeyMap {
-		tabSingleFkeyMethod, iList := g.buildSingleTableRevFkeyFunc(table, rFkey, importList)
+		tabSingleFkeyMethod, iList := g.buildSingleTableRevFkeyFunc(table, rFkey, importList, len(table.RevFKeyMap) > 1)
```

Comment:
That does not take care of the possibility of having two different tables pointing to this table. In that case the RevFKeyMap would still contain more than 1 entries but they won't be from the same table, so that defeats the purpose, no? :-)

Reply:
I added a function to check if we have multiple keys with the same table, and only in that case the flag will be set to true.

PS. Apologies for missing out obvious use cases! Sorry I missed something here as well 😬😭